frontend/src/routes/+layout.svelte

```diff
@@ -24,8 +38,13 @@
     <div class="navbar-end">
       <div class="navbar-item">
         <div class="buttons">
-          <Signup />
-          <Login />
+          {#if !authenticatedValue}
```

Comment:
Svelte has magic syntax to make all this easier - you can simply use the value `$authenticated` and it will automatically update as expected :)